data-raw/watershed/bear_creek/bear_creek.Rmd

```diff
@@ -0,0 +1,58 @@
+---
+title: "Bear Creek"
+author: "[Sadie Gill](mailto:sgill@flowwest.com), [Mark Tompkins](mailto:mtompkins@flowwest.com), [Erin Cain](mailto:ecain@flowwest.com)"
+date: "August 15, 2019"
+output: html_document
+---
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE, message = FALSE, warning = FALSE, out.width = '100%')
+knitr::opts_knit$set(root.dir = rprojroot::find_rstudio_root_file())
+library(tidyverse)
+library(readxl)
+library(cvpiaHabitat)
+```
+## Future Data Improvements
+None Planned 
+
+## Instream Spawning and Rearing Habitat
+**Data Source:** [Calculated with a relationship between flow and mean weighted usable area from watersheds in the region with modeled habitat](http://cvpia-habitat-docs-markdown.s3-website-us-west-2.amazonaws.com/watershed/Regional_Approximation.html)
+```{r, include=FALSE}
+source('data-raw/watershed/regional_approximation/description_regional_approx.R')
+```
+
+`r print_regional_approx('Bear Creek')`
+ 
+```{r}
+source('data-raw/watershed/floodplain_utils.R')
+```
+
+## Floodplain Rearing Habitat
+
+**Data Source:** [Scaled from a Deer Creek flow to floodplain area relationship generated with a 1D HEC-RAS hydraulic model](https://s3-us-west-2.amazonaws.com/cvpiahabitat-r-package/cvpia-sit-model-inputs/DRAFT+Deer+Creek+Hydraulic+Models+Tech+Memo+6-08-07.pdf)
+
+`r print_model_details('Bear Creek', 'fr')`
+
+## Floodplain Data
+```{r}
+# TODO Ask mark about the difference in outcome from his sheet
+bear_creek_floodplain <- scale_fp_flow_area('Bear Creek')
+
+knitr::kable(align = 'c', head(bear_creek_floodplain, 5), caption = "Header Descriptions: flow_cfs = flow in cubic feet per second, FR_floodplain_acres = Fall Run Chinook floodplain acres, SR_floodplain_acres = Spring Run Chinook floodplain acres, ST_floodplain_acres = Steelhead floodplain acres")
```

Comment:
You have included "SR_floodplain_acres" in the header description when there is no spring run, plz remove and be careful with future copy and pastes